<?xml version="1.0"?>
<launch>
  <node pkg="kurt_base" type="kurt_base" name="kurt_base" output="screen">
    <param name="wheel_perimeter" value="0.518" />
    <param name="axis_length" value="0.39" />
    <param name="turning_adaptation" value="0.64" />
  </node>

  <include file="$(find kurt_base)/launch/ekf.launch" />
</launch>
